### ISSUE TITLE
weird bug with long commands

### ISSUE THREAD

**Issue by joaociocca** [gnome-terminator/terminator#373]
tested also with `terminator -g /dev/null` and there's no config file at `$HOME/.config/terminator/config`

Fill out as many of these fields as you can

**Describe the bug**
I feel kinda lost trying to describe this - but it seems that long commands cause a strange behaviour on Terminator that doesn't happen to me on Qt and others...

**To Reproduce**
Open terminator on full screen (for me that gives a 238x49 terminal) and just start typing. When I split the screen vertically, it happens sooner than on a full screen.

**Expected behavior**
that the command be typed normally

**Screenshots**
![image](https://user-images.githubusercontent.com/6473725/105391772-38a12000-5bf9-11eb-9ba8-73dc68c503ec.png)
![image](https://user-images.githubusercontent.com/6473725/105391817-448ce200-5bf9-11eb-85e4-60f59bdd4cc1.png)
![2021-01-21_14h59_55](https://user-images.githubusercontent.com/6473725/105391896-59697580-5bf9-11eb-9e1c-bbf6c587ce28.gif)

**Desktop (please complete the following information):**
Linux kali 5.9.0-kali5-amd64 #1 SMP Debian 5.9.15-1kali1 (2020-12-18) x86_64 GNU/Linux

**Additional context**
I also noticed another problem when going back and forth in history...
![2021-01-21_14h22_41](https://user-images.githubusercontent.com/6473725/105392004-72722680-5bf9-11eb-974d-0c6339aca327.gif)

**Comment by mattrose** [comment]
dupe of #300

**Comment by joaociocca** [comment]
@mattrose oh sorry for that. I'm not used to documenting bugs in things like these so I didn't knew how to describe or keywords to search... I had looked for something like "prompt" but couldn't find anything like it. Qt doesn't use that same library? At least the behaviours I demonstrated on those gifs aren't reproducible there. Still looking for a good alternative, I had started using Qt+tmux, but I keep fighting with mouse/clipboard trouble 😂
anyway, congrats on the amazing project and once again, sorry for the dupe!
